### ISSUE TITLE
z endstop SG90servo probe 

### ISSUE THREAD

**Issue by SrGeek** [KevinOConnor/klipper#762]
[klippy.log](https://github.com/KevinOConnor/klipper/files/2462754/klippy.log)
I am using an SG90 and an opto endstop to home z.  When I home, the servo drops the probe and pulls it back.  I am using https://www.thingiverse.com/thing:2890290 setup.  When I use the setprobe command it pulls the probe up and down no problem.  Got tired of BL Touch.  I havent seen any other configurations with servos other than the BL Touch so I used that one as a start.
i want to add its ramps 1.6 that the servo and endstop share +5v, from the servo source.  the optoendstop sends signal to z-min and uses z-min ground.

**Comment by SrGeek** [comment]
I wired the endstop and servo to their respective power, ground and signal positions on the ramps board.  I queried the probes instead of looking at the query_endstops.  I home the  printer using my hands to set the probe and it works.  Then I tried to home again but it appears that my trigger is not being released after the initial probing.  It goes through some error algorithm and homes my x axis again and ignores homing z.
[klippy.log](https://github.com/KevinOConnor/klipper/files/2462993/klippy.log)
